src/Stryker.CLI/Stryker.CLI/CommandLineConfigHandler.cs

```diff
@@ -0,0 +1,194 @@
+using System.Collections.Generic;
+using System.Linq;
+using McMaster.Extensions.CommandLineUtils;
+using Stryker.Core.Exceptions;
+using Stryker.Core.Options;
+using Stryker.Core.Options.Inputs;
+
+namespace Stryker.CLI
+{
+    public class CliInput
```

Comment:
This whole file is quite big, I would move this class to a separate file as this is a low-hangling readability fruit :D

Reply:
Yeah this is one of these things you overlook after some time